h/views/api/groups.py

```diff
@@ -33,8 +32,8 @@ def groups(request):
         authority=request.params.get("authority"),
         document_uri=request.params.get("document_uri"),
     )
-    all_groups = [GroupContext(group, request) for group in all_groups]
-    all_groups = GroupsJSONPresenter(all_groups).asdicts(expand=expand)
+
+    all_groups = GroupsJSONPresenter(all_groups, request).asdicts(expand=expand)
```

Comment:
This is kind of the story of this PR. Everywhere that wanted to call one of these objects first had to annoyingly make a context object first.